README.md

```diff
@@ -24,9 +24,33 @@ Below is an example of a component that uses the property mixin provided by this
 
 ```js
 import Ember from 'ember'
+import {PropTypes} from 'ember-prop-types'
+
+export default Ember.Component.extend({
+  propTypes: {
+    foo: PropTypes.string,
+    bar: PropTypes.number.isRequired,
+    baz: PropTypes.oneOf([
+      PropTypes.bool,
+      PropTypes.string
+    ])
+  },
+
+  getDefaultProps () {
+    return {
+      foo: 'This is going to be highly profitable'
+    }
+  }
+})
+```
+
+If this mixin is being used in a class other than Component, it will still need to be mixed into the class:
```

Comment:
I would remove the work `still` as someone coming to this repository for the first time won't know the history it being a mixin for components too.